README.md

```diff
@@ -690,6 +690,18 @@ public class Example {
 }
 ```
 
+Default `QeuryMapEncoder` use `Field` property and it does not support field that extend from super class. We recommend that you use `BeanQueryMapEncoder` who will be generated using java beans accessible getter property as query parameter names.
+
+```java
+public class Example {
+  public static void main(String[] args) {
+    MyApi myApi = Feign.builder()
+                 .queryMapEncoder(new BeanQeuryMapEncoder())
+                 .target(MyApi.class, "https://api.hostname.com");
```

Comment:
typo in `BeanQueryMapEncoder`

Reply:
It took me several takes to finally see the typo: "qeury" is supposed to be spelled "query".